docs/specs/validation.yml

```diff
@@ -448,3 +448,37 @@ outputs:
     ["suggestion","duplicate-attribute","Attribute 'description' with value 'description1' is duplicated in 'a.md(3,14)', 'b.md(2,14)'","b.md",2,14]
     ["suggestion","duplicate-attribute","Attribute 'description' with value 'description1' is duplicated in 'a.md(3,14)', 'b.md(2,14)'","a.md",3,14]
 ---
+# Json schema length rule: the length of string must be in range of MinLength and MaxLength
+inputs:
+  docfx.yml: |
+    metadataSchema:
+      - schema.json
+  schema.json: |
```

Comment:
refer to test cases with `markdownValidationRules`, to cover the rule --> JsonSchema conversion logic

Reply:
that's a metadata rule validation, not a content rule, @yufeih